### PR TITLE
Fixup #7554: Inline, fewer LOC, remove redundant definition in cpp.

### PR DESCRIPTION
**Pull request by dok-net**

### CODE REVIEW

**Comment by d-a-v** [comment]
Title should be
> Fixup #7554: ...

**Comment by dok-net** [comment]
@d-a-v totally right, sorry, switched lines on copy&paste.

File: cores/esp8266/Esp.h [hunk 1]
```diff
@@ -123,12 +123,12 @@ class EspClass {
 
 #if defined(F_CPU) || defined(CORE_MOCK)
         constexpr uint8_t getCpuFreqMHz() const
+#else
+        uint8_t getCpuFreqMHz() const
+#endif
```
Comment (devyte) [inline cores/esp8266/Esp.h:128]: Minor nitpick: please put the constexpr specifier on its own line here. It took me several seconds to understand that this is alternates signatures.
E.g.:
```
#if defined(F_CPU) || defined(CORE_MOCK)
        constexpr 
        uint8_t getCpuFreqMHz() const
#else
        uint8_t getCpuFreqMHz() const
#endif
```
or maybe this:
```cpp
#if defined(F_CPU) || defined(CORE_MOCK)
        constexpr 
#endif
        uint8_t getCpuFreqMHz() const
```

Comment (dok-net) [reply cores/esp8266/Esp.h]: You are right.

**Comment by dirkmueller** [comment]
there are out of band inlines in Esp.h at the end. I don't really care which way, but it would be great to stay consistent..

**Review by d-a-v — APPROVED** [review]

**Pull request merged**